docs/source/installation-guide/install-kubernetes.md

```diff
@@ -16,6 +23,28 @@ The Kubernetes deployment of the Gluu Server, also called Cloud Native (CN) Edit
 - [Passport interactions diagram](#architectural-diagram-of-oxpassport)
 - [Jackrabbit interactions diagram](#working-with-jackrabbit)
 
+## Requirmentes for accessing docker images and asset
+
+1. Contact sales@gluu.org for credentials to access and pull our docker images. Existing customers should have recieved the credentials already. If you are using [`pygluu-kubernetes.pyz`](https://github.com/GluuFederation/cloud-native-edition/releases) the tool that parses values.yaml you can skip the next steps. We recommend using helm manually.
+2. Follow the [guide](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-by-providing-credentials-on-the-command-line) to create a secret in your kubernetes environment holding the docker credentials.
+3. Inject the secret name in your [`values.yaml`](https://github.com/GluuFederation/cloud-native-edition/blob/4.5/pygluu/kubernetes/templates/helm/gluu/values.yaml) at `image.pullSecrets` for each service.
+   Forexample:
+   ```yaml
+   ...
+   ...
+   oxauth:
+     image:
+       # -- Image pullPolicy to use for deploying.
+       pullPolicy: IfNotPresent
+       # -- Image  to use for deploying.
+       repository: gluufederation/oxauth
+       # -- Image  tag to use for deploying.
+       tag: 4.5.1-1
+       # -- Image Pull Secrets
+       pullSecrets:
+         - regcred
```

Comment:
The value must be:

```
pullSecrets:
  - name: regcred
```

otherwise it will throw this error:

```
json: cannot unmarshal string into Go struct field PodSpec.spec.template.spec.imagePullSecrets of type v1.LocalObjectReference
```